pyatlan/pkg/utils.py

```diff
@@ -12,6 +12,56 @@
 
 LOGGER = logging.getLogger(__name__)
 
+# Try to import OpenTelemetry libraries
+try:
+    from opentelemetry.exporter.otlp.proto.grpc._log_exporter import (  # type:ignore
+        OTLPLogExporter,
+    )
+    from opentelemetry.sdk._logs import (  # type:ignore
+        LogData,
+        LoggerProvider,
+        LoggingHandler,
+    )
+    from opentelemetry.sdk._logs._internal.export import (  # type:ignore
+        BatchLogRecordProcessor,
+    )
+    from opentelemetry.sdk.resources import Resource  # type:ignore
```

Comment:
Since this feature depends on the external library (`opentelemetry`), could we pin its version in our `requirements.txt`?

Reply:
Got it! 
By default, we don’t provide `opentelemetry` logging support for custom packages (please correct me if I'm wrong)

To enable it, users need to:

1. Install the `opentelemetry` library, which sets `OTEL_IMPORTS_AVAILABLE`.  
2. Import the `add_otel_handler` method and use it like this:  
   ```python
   add_otel_handler(logger, level, resource)
   ```  
   This will attach the new handler. ^^ ✅

Reply:
No @Aryamanz29. This library should not be in the requirements.txt for the pyatlan. That's why the imports are in a try block. Only the workflows need the libraries. They will be in the requirements.txt that is generated for the workflow along with pyatlan.